### ISSUE TITLE
Implement MySQL Storage

### ISSUE THREAD

**Issue by Gnat008** [Gnat008/PerWorldInventory#16]
As the title says: implement storing serialized data in a MySQL database, configurable in the config file.

**Comment by JohOply** [comment]
Is it still scheduled ? :)

**Comment by sgdc3** [comment]
@Gnat008 Good luck ;)

**Comment by Gnat008** [comment]
Yeah this became backburner around the time 1.9 happened. lol. Been trying to work out issues ever since. It will happen at some point, though.
